### PR TITLE
allow for string in search for resType

### PR DESCRIPTION
**Pull request by awirb**

### CODE REVIEW

File: avaframe/com1DFA/com1DFA.py [hunk 1]
```diff
@@ -469,7 +469,7 @@ def prepareInputData(inputSimFiles, cfg):
     relThFile = inputSimFiles['relThFile']
 
     # get dem dictionary - already read DEM with correct mesh cell size
-    demOri = gI.initializeDEM(cfg['GENERAL']['avalancheDir'], cfg['INPUT']['DEM'])
+    demOri = gI.initializeDEM(cfg['GENERAL']['avalancheDir'], demPath=cfg['INPUT']['DEM'])
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:472]: Why is this needed? There are 2 arguments, is this for readability?

Comment (awirb) [reply avaframe/com1DFA/com1DFA.py:472]: yes just for readability

**Review by fso42 — APPROVED** [review]
